src/README.md

```diff
@@ -1,6 +1,6 @@
-# Grafana LLM app (experimental)
+# Grafana LLM app (public preview)
 
-This is a Grafana application plugin which centralizes access to LLMs across Grafana.
+This Grafana application plugin centralizes access to LLMs across Grafana.
 
 It is responsible for:
```

Comment:
Think we need to add vector search features in this section still

Reply:
We will -- I left it out just while all those pieces are landing, and figured we'll update the docs again when ready.

Reply:
Discussion in Slack about docs & separating these 2 releases / iterations